### PR TITLE
Added an additional validation property to ucrInput control

### LINKED ISSUE #5762: Validation of numeric expressions in input controls
At the moment a custom control like ucrTextbox can validate if a value is numeric or not, but if I type in , for instance, 80/2 which results to 40, it is unable to recognize this as an expression which could result to a numeric expression.  

For this kind of validation the current control needs to be enhanced.

### PR DESCRIPTION
**Pull request by Wycklife**
Fixes #5762

In progress

### CODE REVIEW

File: instat/ucrInput.vb [hunk 1]
```diff
@@ -26,6 +26,7 @@ Public Class ucrInput
     Protected dcmMinimum As Decimal = Decimal.MinValue
     Protected dcmMaximum As Decimal = Decimal.MaxValue
     Protected bMinimumIncluded, bMaximumIncluded As Boolean
+    Protected bExpressionAllowed As Boolean = True
```
Comment (maxwellfundi) [inline instat/ucrInput.vb:29]: Why do we have this variable here. What does it do?

**Comment by shadrackkibet** [comment]
@Wycklife what is the state of this?

**Comment by Patowhiz** [comment]
@Wycklife as discussed in Skype will work on this and do a PR to your branch.

**Comment by Patowhiz** [comment]
@Wycklife I have pushed the code additions to PR #5826. This PR can now be closed.

**Comment by Wycklife** [comment]
@africanmathsinitiative/developers I'm closing this since there is another  PR #5826 on the same.

**Pull request closed without merging**